data/Templates/Hl7v2/ID/_RelatedPerson.liquid

```diff
@@ -0,0 +1,30 @@
+﻿{% if NK1 -%}
+{% if NK1.12.1 and NK1.12.4 -%}
+{% capture segment -%}{{NK1.12.1.Value}},{{NK1.12.4.Value}}{% endcapture -%}
+{% capture input -%}{{segment | generate_id_input: 'RelatedPerson', true, baseId}}{% endcapture -%}
+{% else -%}
+{% capture input -%}{{NK1.Value | generate_id_input: 'RelatedPerson', true, baseId}}{% endcapture -%}
+{% endif -%}
+{% elsif PID -%}
+{% if PID.21 -%}
+{% capture segment -%}{{PID.21.1.Value}},{{PID.21.4.Value}}{% endcapture -%}
```

Comment:
We'd better also ensure the existence of `PID.21.1` and `PID.21.4` like `{% if NK1.12.1 and NK1.12.4 -%}` for consistency :)

Reply:
Thanks for pointing this out. It is fixed in the update.